package.json

```diff
@@ -79,27 +79,6 @@
         "deepmerge": "^3.3.0",
         "semver": "^6.3.0"
     },
-    "peerDependencies": {
-        "autoprefixer": "^9.7.0",
-        "babel-loader": "^8.0.5",
-        "clean-webpack-plugin": "^2.0.1",
-        "copy-webpack-plugin": "^5.0.2",
-        "css-loader": "^3.2.0",
-        "cssnano": "^4.1.10",
-        "file-loader": "^3.0.1",
-        "jquery": "^3.4.0",
-        "mini-css-extract-plugin": "^0.8.0",
-        "postcss-loader": "^3.0.0",
-        "resolve-url-loader": "^3.1.0",
-        "sass-loader": "^7.3.1",
-        "typescript": "^3.6.3",
-        "webpack": "^4.41.2",
```

Comment:
Zou webpack hier wel laten staan, deze valideer je ook nergens en is ook wel de basis van heel de builder.

Reply:
ok, zal die weer toevoegen.

Reply:
https://github.com/freshheads/webpack-config-builder/pull/43/commits/722fe74eec0cc243f3da7aae4c6329a73d29c11a